<launch>

<node name="osmosis_control_node" type="osmosis_control_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="teleop_node" type="teleop_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e" />
<node name="safety_pilot_node" type="safety_pilot_node" pkg="osmosis_control" launch-prefix="xterm -e" />
<node name="localization_node" type="localization_node" pkg="osmosis_control" />
<node name="graph_planner_node" type="graph_planner_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="joy_teleop_node" type="joy_teleop_node" pkg="osmosis_control" output="screen"/>
<node name="mission_manager_node" type="mission_manager_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="HMI_node" type="HMI_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>
<node name="joy_node" type="joy_node" pkg="joy" output="screen">
	<param name="deadzone" type="double" value="0.13" />
</node>

<node name="fault_injection_node" type="fault_injection_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/>


<!--node name="navigation_node" type="navigation_node" pkg="osmosis_control" output="screen" launch-prefix="xterm -e"/-->

</launch>
